src/careamics/models/lvae/likelihoods.py

```diff
@@ -355,6 +356,8 @@ def log_likelihood(self, x: torch.Tensor, params: dict[str, torch.Tensor]):
         torch.Tensor
             The log-likelihood tensor. Shape is (B, C, [Z], Y, X).
         """
+        self.data_mean = self.data_mean.to(x.device)
+        self.data_std = self.data_std.to(x.device)  # TODO this is ugly fix !!
```

Comment:
This can stay temporarily, but would be changed soon since it is not very clean in my opinion.

Reply:
This comes from previous pr and we discussed it should change